src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -282,10 +282,16 @@ def evaluate(
     :keyword evaluators: Evaluators to be used for evaluation. It should be a dictionary with key as alias for evaluator
         and value as the evaluator function.
     :paramtype evaluators: Optional[Dict[str, Callable]
-    :keyword evaluator_config: Configuration for evaluators. The configuration should be a dictionary with evaluator
-        names as keys and a dictionary of column mappings as values. The column mappings should be a dictionary with
-        keys as the column names in the evaluator input and values as the column names in the input data or data
-        generated by target.
+    :keyword evaluator_config: Configuration for evaluators. The configuration should be a dictionary
+        with evaluator names as keys and a dictionary of their input column mappings as values.
+        The input column mappings should be a dictionary with column as parameters of callable evaluators,
+        and values as columns in input data or the output dictionaries generated by target.
+        For example, evaluator_config = {"question": "${data.chat_input"} means that
+        "question" is a required parameter for evaluator, and "chat_input" a column in input data.
+        Similarly, if the key "chat_input" is in the output dictionary of target callable,
+        use "${target.chat_input"}. Users only need to specify this mapping
+        if the columns are different. For example, {"question": "${data.question"}
+        or {"question": "${target.question}"} is redundant.

```

Comment:
Can we say something along the lines of it is optional and required only if data columns have different names than what evaluator names